src/main/scala/uk/gov/hmrc/play/bootstrap/config/CryptoValidation.scala

```diff
@@ -17,10 +17,9 @@
 package uk.gov.hmrc.play.bootstrap.config
 
 import javax.inject.Inject
-
 import play.api.Configuration
-import uk.gov.hmrc.crypto.ApplicationCryptoDI
+import uk.gov.hmrc.crypto.ApplicationCrypto
 
 class CryptoValidation @Inject()(configuration: Configuration) {
```

Comment:
you could also inject `ApplicationCrypto` directly instead of `Configuration`

Reply:
Done.